src/scene/scene.js

```diff
@@ -200,14 +201,22 @@ class Scene extends EventHandler {
         }
     }
 
+    get skyboxEnabled() {
```

Comment:
I'm not sure about the naming here. If a user does `scene.skyboxEnabled = false;`, they might expect it to have the same effect as having no skybox at all. But this just disables _rendering_ of the skybox - it doesn't disable the skybox as a source for IBL, right?

Reply:
Yeah it's unfortunate that scene conflates skybox and lighting data. In the scene, skybox means both the background image as well as the IBL data. Ideally the scene would have a 'skybox' and 'lighting data' and they should be two separate things.

Re naming, 'renderSkybox' sounds to me like it's going to render a skybox :) Can't really think of a better name though.